### ISSUE TITLE
clean up support comments in the compat branch

### ISSUE THREAD

**Issue by mzgol** [jquery/jquery#1710]
Originally reported by @dmethvin: http://bugs.jquery.com/ticket/15141

Not sure if it's practical to cherry-pick, and there will be several additional code blocks that need cleanup.

Ref  https://github.com/jquery/jquery/commit/d837f119c3729565103005d5d7fa89e1dd8110cb

**Comment by dmethvin** [comment]
At this point it's been done as well in compat as it needs to be done before it rides off into the sunset.

**Comment by mzgol** [comment]
@dmethvin Please remember to remove a milestone & add a "wontfix" label when closing a bug as "won't fix" :)
